### ISSUE TITLE
Somof 3DPW train set and test set

### ISSUE THREAD

**Issue by young0304** [evendrow/somoformer#4]
Thank you for your great work. Are the experimental results in VIM on the SoMoF 3DPW trained on the 3DPW test set and tested only on the training set? Looking forward to your reply.
